lxd/instance/drivers/driver_qemu.go

```diff
@@ -7464,6 +7464,9 @@ func (d *qemu) freeVsockID(vsockID uint32) bool {
 			// The syscall to the vsock device returned "no such device".
 			// This means the address (Context ID) is free.
 			return true
+		} else {
```

Comment:
We generally use the "happy path" style in LXD, see

https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

So this would be written like:

```go
// freeVsockID returns true if the given vsockID is not yet acquired.
func (d *qemu) freeVsockID(vsockID uint32) bool {
	c, err := lxdvsock.Dial(vsockID, shared.HTTPSDefaultPort)
	if err != nil {
		var unixErrno unix.Errno

		if !errors.As(err, &unixErrno) {
			return false
		}

		if unixErrno != unix.ENODEV {
             // Skip the vsockID if another syscall error was encountered.
			return false
        }

	    // The syscall to the vsock device returned "no such device".
		// This means the address (Context ID) is free.
		return true
	}

	// Address is already in use.
	c.Close()
	return false
}
```

Reply:
Didn't know there is a term for this :) Thanks, changed.